<launch>

	<node name="modem_publisher_0" pkg="rospyd7a" type="modem_publisher.py">
		<param name="dash7_port" type="str" value="/dev/ttyUSB0"/>
		<param name="dash7_baudrate" type="int" value="115200"/>
		<param name="x_coor" type="double" value="0.0"/>
		<param name="y_coor" type="double" value="0.16"/>
		<param name="z_coor" type="double" value="0.43"/>
	</node>

</launch>
